### ISSUE TITLE
[ISSUE] fix Inputbox component that have any types

### ISSUE THREAD

**Issue by wit03** [cpe-kmutt-student/bangmod-hackathon-2023#21]
@Encall There is ReactEvent or something that will match the parameter "e" types please check it out

**Comment by riflowth** [comment]
Fixed on #22

**Comment by riflowth** [comment]
Related doc: https://preactjs.com/guide/v10/differences-to-react/#use-oninput-instead-of-onchange

```ts
const handleChange = (event: Event) => {
    if (!(event.target instanceof HTMLInputElement)) return;
    const value = event.target.value;

    // Do whatever you want with 'value' variable !!!
  };

  return (
    <input
      onInput={handleChange}
    />
  );
```
